java/configuration/src/main/java/sleeper/configuration/properties/instance/UserDefinedInstancePropertyImpl.java

```diff
@@ -113,6 +115,24 @@ public boolean isIgnoreEmptyValue() {
         return ignoreEmptyValue;
     }
 
+    public InstanceProperty getDefaultProperty() {
+        return defaultProperty;
+    }
+
+    @Override
+    public String computeValue(String value, InstanceProperties instanceProperties) {
+        if (value != null) {
+            return value;
+        } else {
```

Comment:
We could avoid the nested block by removing the else clause here.

Reply:
Addressed